### PR TITLE
blocks, block entities, and game events

### PR DESCRIPTION
**Pull request by eristhea**
thought we should get up to speed on this

### CODE REVIEW

**Review by EnnuiL — APPROVED** [review]
I approve of the progress so far!

File: mappings/net/minecraft/block/RedstoneWireBlock.mapping [hunk 1]
```diff
@@ -18,6 +18,8 @@ CLASS net/minecraft/unmapped/C_hbdlqish net/minecraft/block/RedstoneWireBlock
 	FIELD f_tfkeexfj SHAPES Ljava/util/Map;
 	FIELD f_vvlqyglw SHAPES_UP Ljava/util/Map;
 	FIELD f_vvxqbtel WEST I
+	METHOD m_ahfsctfo ([Lnet/minecraft/unmapped/C_vgpupfxx;)V
+		ARG 0 positions
```
Comment (EnnuiL) [inline mappings/net/minecraft/block/RedstoneWireBlock.mapping:22]: hm, i just realized that you can't really make a plural of `pos`; maybe `poses`, `posi`? :p
yeah, keep it as is

**Review by EnnuiL — APPROVED** [review]

File: mappings/net/minecraft/block/PitcherCropBlock.mapping [hunk 2]
```diff
@@ -1,3 +1,16 @@
 CLASS net/minecraft/unmapped/C_fntbpikr net/minecraft/block/PitcherCropBlock
+	FIELD f_cjuojjbg FULLY_GROWN_UPPER_SHAPE Lnet/minecraft/unmapped/C_zscvhwbd;
+	FIELD f_fnokzkbm BONEMEAL_AGE_INCREASE I
```
Comment (EnnuiL) [inline mappings/net/minecraft/block/PitcherCropBlock.mapping]:
```suggestion
	FIELD f_fnokzkbm BONE_MEAL_AGE_INCREASE I
```

File: mappings/net/minecraft/block/TorchflowerCropBlock.mapping [hunk 3]
```diff
@@ -1,3 +1,6 @@
 CLASS net/minecraft/unmapped/C_ujhirzat net/minecraft/block/TorchflowerCropBlock
+	FIELD f_epidvlrx BOX_OFFSET F
+	FIELD f_nexibmib BONEMEAL_AGE_INCREASE I
```
Comment (EnnuiL) [inline mappings/net/minecraft/block/TorchflowerCropBlock.mapping]:
```suggestion
	FIELD f_nexibmib BONE_MEAL_AGE_INCREASE I
```

**Review by EnnuiL — CHANGES_REQUESTED** [review]

**Review by EnnuiL — APPROVED** [review]

**Review by OroArmor — APPROVED** [review]

**Pull request merged**